### PR TITLE
Upgrading ember-cli to 1.13.14 along with deps

### PR DESCRIPTION
**Pull request by jasonmit**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   "name": "ember-cli-simple-store",
   "dependencies": {
-    "ember": "1.13.7",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-qunit": "0.4.9",
-    "ember-qunit-notifications": "0.0.7",
-    "ember-resolver": "~0.1.18",
+    "ember": "1.13.11",
```
Comment (jasonmit) [inline bower.json:4]: Used the bower deps + criteria that ships with ember-cli 1.13.14

**Comment by toranb** [comment]
@jasonmit you might need to run ember-init also as the resolver import changed from a slash to a dash (going from 1.13.8 to 1.13.13/14)

**Comment by jasonmit** [comment]
@toranb I think that's part of ember-cli 2.0.0 since it still is a bower dep in 1.13.14 and not a npm dep.

#master
https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json#L38

1.13.14:
https://github.com/ember-cli/ember-cli/blob/v1.13.14/blueprints/app/files/package.json

Edit: ran `ember init` and the resolver changes were not brought over so looks like it's only a 2.x thing.  We should upgrade once it's out of beta.

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```
Comment (jasonmit) [inline LICENSE.md:1]: @toranb are you fine with this?  README says MIT at the bottom

Comment (toranb) [inline LICENSE.md:1]: I assume that's cool - MIT is what I normally rock

**Pull request merged**